src/DynamoCoreWpf/UI/GuidedTour/dynamo_guides.json

```diff
@@ -965,6 +964,110 @@
 						"Action": "execute"
 					}
 				]
+			},
+			{
+				"Sequence": 3,
+				"Name": "placeNode",
+				"Type": "tooltip",
+				"ShowLibrary": true,
+				"TooltipPointerDirection": "none",
+				"Width": 480,
+				"Height": 248,
+				"StepContent": {
+					"Title": "OnboardingPlaceNodeTitle",
+					"FormattedText": "OnboardingPlaceNodeText"
+				},
+				"HostPopupInfo": {
+					"VerticalPopupOffset": 300,
+					"HorizontalPopupOffset": 100,
+					"PopupPlacement": "right",
+					"HostUIElementString": "sidebarGrid",
+					"WindowName": "LibraryView",
+					"HighlightRectArea": {
+						"HighlightColor": "#EF9362",
+						"UIElementTypeString": "WebBrowser",
+						"UIElementGridContainer": "sidebarGrid",
+						"WindowElementNameString": "ByCenterPointRadius",
+						"WindowName": "LibraryView"
+					},
+					"CutOffRectArea": {
+						"WidthBoxDelta": 0,
+						"HeightBoxDelta": 0
+					}
+				},
+				"UIAutomation": [
+					{
+						"Sequence": 0,
+						"ControlType": "JSFunction",
+						"WindowName": "LibraryView",
+						"Name": "InvokeJSFunction",
+						"Action": "execute",
+						"JSFunctionName": "collapseExpandPackage",
+						"JSParameters": [ "Geometry", "LibraryItemText" ]
+					},
+					{
+						"Sequence": 1,
+						"ControlType": "JSFunction",
+						"WindowName": "LibraryView",
+						"Name": "InvokeJSFunction",
+						"Action": "execute",
+						"JSFunctionName": "collapseExpandPackage",
+						"JSParameters": [ "Curves", "LibraryItemGroupText" ]
+					},
+
+					{
+						"Sequence": 2,
+						"ControlType": "JSFunction",
+						"WindowName": "LibraryView",
+						"Name": "InvokeJSFunction",
+						"Action": "execute",
+						"JSFunctionName": "collapseExpandPackage",
+						"JSParameters": [ "Circle", "LibraryItemGroupText" ]
+					},
+					{
+						"Sequence": 4,
+						"ControlType": "JSFunction",
+						"Name": "InvokeJSFunction",
+						"JSFunctionName": "getDocumentClientRect",
+						"JSParameters": [ "ByCenterPointRadius" ],
+						"Action": "execute"
+					},
+					{
+						"Sequence": 5,
+						"ControlType": "button",
+						"WindowName": "PopupWindow",
+						"Name": "NextButton",
+						"JSFunctionName": "",
+						"Action": "disable"
+					},
+					{
+						"Sequence": 3,
+						"ControlType": "function",
+						"Name": "CreateNode",
+						"JSFunctionName": "",
+						"JSParameters": [ "Circle.ByCenterPointRadius" ],
+						"Action": "execute",
+						"ExecuteCleanUpForward": false,
+						"ExecuteCleanUpBackward": true
+					}
+				]
+			},
+			{
+				"Sequence": 4,
+				"Name": "connectNodes",
+				"Type": "tooltip",
+				"ShowLibrary": false,
+				"Width": 480,
+				"Height": 320,
+				"StepContent": {
+					"Title": "OnboardingGuideWelcomeTitle",
+					"FormattedText": "GetStartedGuideWelcomeText"
```

Comment:
I am touching this in my PR so not sure if this should be updated as well

Reply:
Oh, I've added this step just for mocking. Do you want me to remove it?